dashboard/package.json

```diff
@@ -9,7 +9,7 @@
     "build-only": "vite build",
     "type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
     "preview": "vite preview",
-    "lint": "prettier --check .",
+    "lint": "eslint .",
```

Comment:
Is Prettier ran through ESList? We want both, right?

Reply:
Prettier is called with `npm run format` (the line below lint). I think this is correct - prettier is a code formatter, eslint is a code linter.

Reply:
Yeah, sounds good to me. We need to add `format` to the dashboard workflow then as lint is already there. But it won't be the same using `prettier --write .`; maybe add a new `format-check` that runs `prettier --check .`, and run that one in CI.

Reply:
I'll add a "format-check" npm script that runs `prettier --check` for use in our CI.